modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,385 @@
+// APP DETAIL REQUESTS: Requests tab
+//tag::app-detail-requests[]
+Inbound Request Volume::
+Inbound request count total, which you can group by response time or protocol 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Outbound Request Volume::
+Outbound request count total, which you can group by response time, protocol, or by the 20 hosts (such as an Object Store or a `cloudhub.io` address) with the most outbound requests 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: requests -> Requests (SATHISH)
+Top Slowest Requests by Path::
+The 20 slowest endpoints in this app
++
+The requests must take more than 25ms. If no data is available, then no requests take more than 25ms to process.
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Request Latency by Endpoint::
+Inbound or outbound latency, in milliseconds, by endpoint
++
+The table captures the average latency and the latency value of requests at the p50, p75, p90, p95, and p99 percentiles.
+ 
+//end::app-detail-requests[]
+//-----------------------//
+//
+//-----------------------//
+//  APP DETAIL ERRORS: Errors tab
+//tag::app-detail-errors[]
+Failed Outbound Request Volume::
+Total number of failed outbound requests at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Top 20 Failed Requests by Total Count::
+Total number of inbound or outbound requests to the 20 endpoints with the most failed requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Top 20 Failed Outbound Requests by Total Count::
+Total number of unsuccessful outbound requests to the 20 endpoints with the most failed requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-errors[]
+//-----------------------//
+//
+//-----------------------//
+// APP DETAIL JVM: JVM tab
+//tag::app-detail-jvm[]
+//TODO: Old Gen GC Count -> Old Generation Garbage Collection Count (SATHISH)
+Old Gen GC Count::
+Total number of times old generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Old Gen GC Duration -> Old Generation Garbage Collection Duration (SATHISH)
+Old Gen GC Duration::
+Amount of time taken for old generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Young Gen GC Count -> Young Generation Garbage Collection Count (SATHISH)
+Young Gen GC Count::
+Total number of times young generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Young Gen GC Duration -> Young Generation Garbage Collection Duration (SATHISH)
+Young Gen GC Duration::
+Amount of time taken for young generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Committed::
```

Comment:
All this stuff reminds me of stuff in [RTM](https://docs.mulesoft.com/runtime-manager/monitoring-dashboards). Are they the same? If so, can we consolidate?

Reply:
Can look into it at some point after release. I noticed some charts with the same names after the fact, but same name may not mean same data. There are likely some overlapping charts, but we'd have to know if the data that the tables are capturing is from the same source (and/or that the values couldn't differ based on the ways it's aggregated, etc.). Not something we can do now though.

Reply:
TODO (LATER): @dukesphere (create wi)